packages/v4/patternfly-docs/content/design-guidelines/demos/primary-detail/primary-detail.md

```diff
@@ -8,7 +8,10 @@ A **primary-detail layout** is an interface that shows a list of items and the c
 ## Elements
 A primary component sits in a container on the page, and detail components typically live in a slide-out [drawer](/components/drawer/react).
```

Comment:
Can you update this "drawer" link - the URL has changed to not have the "react" part in it anymore!